client/views/windows/sign_in.tpl

```diff
@@ -1,23 +1,18 @@
 <form class="container" method="post" action="">
```

Comment:
I recommend https://github.com/thelounge/thelounge/pull/2199/files?w=1 for this file.